armbian/base/build/customize-armbian-rockpro64.sh

```diff
@@ -317,6 +325,7 @@ testnet=1
 # server
 server=1
 listen=1
+listenonion=1
```

Comment:
(Optional) Not necessarily for this PR, but perhaps we should also update the docs accordingly, since we have an annotated `bitcoin.conf` there?